bundles/org.openhab.binding.neohub/README.md

```diff
@@ -33,21 +33,31 @@ It signs on to the hub using the supplied connection parameters, and it polls th
 The NeoHub supports two Application Programming Interfaces "API" (an older "legacy" one, and a modern one), and this binding can use either of them to communicate with it.
 Before the binding can communicate with the hub, the following Configuration Parameters must be entered.
 
-| Configuration Parameter | Description                                                                                 |
-|-------------------------|---------------------------------------------------------------------------------------------|
-| hostName                | Host name (IP address) of the NeoHub (example 192.168.1.123)                                |
-| portNumber              | Port number of the NeoHub (Default=4242)                                                    |
-| pollingInterval         | Time (seconds) between polling requests to the NeoHub (Min=4, Max=60, Default=60)           |
-| socketTimeout           | Time (seconds) to allow for TCP socket connections to the hub to succeed (Min=4, Max=20, Default=5) |
-| preferLegacyApi         | ADVANCED: Prefer the binding to use older API calls; if these are not supported, it switches to the new calls (Default=false) |
+| Configuration Parameter    | Description                                                                                              |
+|----------------------------|----------------------------------------------------------------------------------------------------------|
+| hostName                   | Host name (IP address) of the NeoHub (example 192.168.1.123)                                             |
+| useWebSockets<sup>1)</sup> | Use secure WebSockets to connect to  the NeoHub (example `true`)                                         |
```

Comment:
Here and everywhere else: Shouldn't it be [WebSocket](https://en.wikipedia.org/wiki/WebSocket) (singular) as the name of the protocol?

Reply:
Insofar as there is only one session open at any time, I suppose there is only one TCP- or Web-Socket in use at any time. However over time, the binding will create new Xxx- sockets as needed whenever a prior Xxx socket is dropped e.g. due to a communication error.

Reply:
Ok. Done.